tasks/connectors/edgescan/README.md

```diff
@@ -33,6 +42,76 @@ These are some quick examples:
 | kenna_api_host     | false    | Kenna API hostname                                                           | api.us.kennasecurity.com |
 | output_directory   | false    | The task will write JSON files here (path is relative to the base directory) | output/edgescan          |
 | create_findings    | false    | The task will create findings, instead of vulnerabilities                    | false                    |
+| network_vulns      | false    | The task will include network layer vulnerabilities                          | true                     |
+| application_vulns  | false    | The task will include application layer vulnerabilities                      | true                     |
+
+## Data Mappings
+
+Edgescan assets do not map directly to Kennna assets due to Edgescan assets being more flexible in their definition.
+Edgescan location specifiers and hosts are more like Kenna assets. Location specifiers define the location and hosts hold extra information.
+Not all location specifiers have a host, and not all vulnerabilities have a directly related location specifier.
```

Comment:
Why don't vulnerabilities have a location? If we know the system's hostname or IP address, don't we know the vulnerability's location?